spec/test-helpers.js

```diff
@@ -10,6 +10,15 @@ const TestHelpers = {
     /** Length of example audio clip (in seconds) */
     EXAMPLE_FILE_DURATION: 21,
 
+    /** Example audio clip with waveform */
+    EXAMPLE_2_FILE_PATH: '/base/spec/support/stereo.mp3',
```

Comment:
can you rename this to `EXAMPLE_STEREO_FILE_PATH` and change the other names too?